decidim-core/app/presenters/decidim/home_stats_presenter.rb

```diff
@@ -76,7 +76,7 @@ def render_stats_data(name, data)
 
     def public_participatory_spaces
       @public_participatory_spaces ||= Decidim.participatory_space_manifests.flat_map do |manifest|
-        manifest.participatory_spaces.call(organization).public_spaces
+        manifest.participatory_spaces.call(organization).public_spaces.try(:non_private_spaces)
```

Comment:
Why adding a `try` here? I don't understand how `public_spaces` isn't synonym of `non_private_spaces`.

Reply:
@mrcasals says that
> Models implementing the Decidim::Participable concern are not required to include the Decidim::HasPrivateUsers one, so they might not have the private_space column and this will raise an error.

therefore I understand that `non_provate_spaces` needs to be define in `HasPrivateUsers` and this will not be for all `ParticipatorySpaces` no?

Reply:
If we override it the way we do down below, this should be unnecessary. Still, with the current implementation, `try` will raise `NoMethodError` if `non_private_spaces` isn't defined.

Reply:
requested changes applied!